cmake/cuda-api-wrappersConfig.cmake.in

```diff
@@ -0,0 +1,9 @@
+# Get the directory containing this file.
```

Comment:
So, why do we need this `.in` file, if we have `EXPORT()` commands in the CMakeLists.txt?

Reply:
This can be handy to define a different `cuda-api-wrappersConfig.cmake` file for `install` or `build` target. In that file we must state the dependencies of `cuda-api-wrappers` since they are not automatically written into the `cuda-api-wrappersTarget.cmake` file. I wanted to write the dependency on `libcudart` here, but to have a clean solution I should also write a find-module for `libcudart`, and I did not have enough time for that.